### PR TITLE
cloud: support built-in monitoring for serverless tiers

### PR DESCRIPTION
**Pull request by qiancai**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by lilyjazz — APPROVED** [review]

**Review by ran-huang — APPROVED** [review]

**Review by lilyjazz — APPROVED** [review]

File: tidb-cloud/built-in-monitoring.md [hunk 1]
```diff
@@ -32,6 +28,10 @@ All monitoring data is kept for 2 days.
 
 The following sections illustrate the metrics on the Monitoring page.
 
+> **Note:**
+>
+> For [Serverless Tier](/tidb-cloud/select-cluster-tier.md#serverless-tier-beta) clusters, the Monitoring page only provides only a subset of monitoring metrics, which are marked with `*` in the **Metric name** column of the following sections.
```
Comment (qiancai) [inline tidb-cloud/built-in-monitoring.md:33]:
```suggestion
> For [Serverless Tier](/tidb-cloud/select-cluster-tier.md#serverless-tier-beta) clusters, the Monitoring page only provides a subset of monitoring metrics, which are marked with `*` in the **Metric name** column of the following sections.
```

File: tidb-cloud/built-in-monitoring.md [hunk 2]
```diff
@@ -32,6 +28,10 @@ All monitoring data is kept for 2 days.
 
 The following sections illustrate the metrics on the Monitoring page.
 
+> **Note:**
+>
+> For [Serverless Tier](/tidb-cloud/select-cluster-tier.md#serverless-tier-beta) clusters, the Monitoring page only provides a subset of monitoring metrics, which are marked with `*` in the **Metric name** column of the following sections.
```
Comment (ran-huang) [inline tidb-cloud/built-in-monitoring.md:33]: How about this? 
```suggestion
> For [Serverless Tier](/tidb-cloud/select-cluster-tier.md#serverless-tier-beta) clusters, the Monitoring page only provides a subset of monitoring metrics, which are marked with `*` in the **Metric name**.
```

**Review by ran-huang — COMMENTED** [review]
Rest LGTM

File: tidb-cloud/built-in-monitoring.md (hunk 2 above)
Comment (qiancai) [reply tidb-cloud/built-in-monitoring.md:33]: As the actual metric name does not include `*` in it, can we keep "in the **Metric name** column" and remove "of the following sections"?

Comment (ran-huang) [reply tidb-cloud/built-in-monitoring.md:33]: Sure. Updated.

File: tidb-cloud/built-in-monitoring.md [hunk 3]
```diff
@@ -44,23 +44,23 @@ The following sections illustrate the metrics on the Monitoring page.
...
+| `*` Query Duration | avg-{SQL Types}, 99-{SQL Types} | The duration from receiving a request from the client to TiDB till TiDB executing the request and returning the result to the client. In general, client requests are sent in the form of SQL statements; however, this duration can include the execution time of commands such as `COM_PING`, `COM_SLEEP`, `COM_STMT_FETCH`, and `COM_SEND_LONG_DATA`. TiDB supports Multi-Query, which means the client can send multiple SQL statements at one time, such as `select 1; select 1; select 1;`. In this case, the total execution time of this query includes the execution time of all SQL statements. |
```
Comment (qiancai) [inline tidb-cloud/built-in-monitoring.md:63]:
```suggestion
| `*` Average Query Duration | avg-{SQL Types}, 99-{SQL Types} | The average duration from receiving a request from the client to TiDB till TiDB executing the request and returning the result to the client. In general, client requests are sent in the form of SQL statements; however, this duration can include the execution time of commands such as `COM_PING`, `COM_SLEEP`, `COM_STMT_FETCH`, and `COM_SEND_LONG_DATA`. TiDB supports Multi-Query, which means the client can send multiple SQL statements at one time, such as `select 1; select 1; select 1;`. In this case, the total execution time of this query includes the execution time of all SQL statements. |
```

File: tidb-cloud/built-in-monitoring.md [hunk 4]
```diff
@@ -71,8 +71,8 @@ The following sections illustrate the metrics on the Monitoring page.
 
 | Metric name  | Labels | Description                                   |
 | :------------| :------| :-------------------------------------------- |
-| Transaction Per Second | {types}-{transaction model} | The number of transactions executed per second. |
-| Transaction Duration | avg-{transaction model}, 99-{transaction model} | The execution duration of a transaction. |
+| `*` Transaction Per Second | {types}-{transaction model} | The number of transactions executed per second. |
+| `*` Transaction Duration | avg-{transaction model}, 99-{transaction model} | The execution duration of a transaction. |
```
Comment (qiancai) [inline tidb-cloud/built-in-monitoring.md:75]:
```suggestion
| `*` Average Transaction Duration | avg-{transaction model}, 99-{transaction model} | The average execution duration of a transaction. |
```

**Pull request merged**